### PR TITLE
Use precomputed mean/std to compute skewness and kurtosis

### PR DESCRIPTION
**Pull request by MaxNoe**
master:
```
334 µs ± 1.48 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)
```

this:
```
55.7 µs ± 549 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

### CODE REVIEW

File: ctapipe/image/statistics.py [hunk 1]
```diff
@@ -1,17 +1,77 @@
...
 
 from ..containers import StatisticsContainer
 
 
+def skewness(data, mean=None, std=None):
+    '''Calculate skewnewss (normalized third central moment)
+    with allowing precomputed mean and std.
+
+    With precomputed mean and std, this is ~10x faster than scipy.stats.skew
+    for our use case (1D arrays woth ~100-1000 elements)
```
Comment (dneise) [inline ctapipe/image/statistics.py:11]: typo in with

File: ctapipe/image/statistics.py [hunk 2]
```diff
@@ -1,17 +1,77 @@
...
+    std = std or np.std(data)
+    return np.average(((data - mean) / std)**3)
+
+
+def kurtosis(data, mean=None, std=None, fisher=True):
+    '''Calculate kurtosis (normalized fourth central moment)
+    with allowing precomputed mean and std.
+
+    With precomputed mean and std, this is ~10x faster than scipy.stats.skew
+    for our use case (1D arrays woth ~100-1000 elements)
```
Comment (dneise) [inline ctapipe/image/statistics.py:37]: same typo again

**Comment by watsonjj** [comment]
These are simple mathematical operations and has couple of temporary arrays....you could check what further performance you get with `njit`, and maybe with `(fastmath=True)`

**Comment by MaxNoe** [comment]
I tried with njit and the improvement if any is in the noise.

**Comment by MaxNoe** [comment]
Added more tests and njit, since it gave a 10 % improvement. Pls. review again.

**Comment by MaxNoe** [comment]
@kosack done

**Comment by MaxNoe** [comment]
pytest cov seems to be unable to work with the njit functions. This is unfortunate but also not really surprising in hindsight.

This is way diff coverage is so low

**Pull request merged**